plugins/toolkit/jetbrains-core/src-233/software/aws/toolkits/jetbrains/core/explorer/ActionToolbarLayoutUtil.kt

```diff
@@ -0,0 +1,9 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.core.explorer
+
+import com.intellij.openapi.actionSystem.ActionToolbar
+
+fun setToolbarLayout(toolbar: ActionToolbar) {
+    toolbar.layoutPolicy = ActionToolbar.NOWRAP_LAYOUT_POLICY
+}
```

Comment:
`ToolbarLayoutStrategy` doesn't exist in 233, so it would be less overall work if we just aliased
`ToolbarLayoutStrategy.NOWRAP_STRATEGY to be ActionToolbar.NOWRAP_LAYOUT_POLICY` i n233